pkg/alertmanager/README.md

```diff
@@ -0,0 +1,108 @@
+## Alertmanager Package
+
+The alertmanager package and CRDs together make up the alertmanager integration in the prometheus operator. Here is a
```

Comment:
```suggestion
The Alertmanager package and CRDs together make up the Alertmanager integration in the Prometheus-Operator. Here is a
```

Reply:
Another way to say is that the pkg/alertmanager package implements the controller reconciling Alertmanager  and AlertmanagerConfig resources in a cluster.